jarviscli/plugins/calculator.py

```diff
@@ -0,0 +1,292 @@
+import tkinter as tk
```

Comment:
Hmm, tkinter may not be installed. We should add some kind of check if this import fails (I'll think of something).

Have some graphical glitches - probably because my i3 desktop is messing with windows size... I'll check with a "regular" window manger later ;).

First look code looks good to me.